### ISSUE TITLE
deprecated?

### ISSUE THREAD

**Issue by jonnymangothinkjam** [mrdotb/facebook-api-video-upload#21]
Is this deprecated?

I keep getting a 100 error when sending a video, and from what i can tell i have the right api access, unless i've missed one

**Comment by mrdotb** [comment]
HTTP 100 is not an error code
I will check if it's still working.

**Comment by jonnymangothinkjam** [comment]
the HTTP error is 400, the 100 is coming back from FB if it helps

{ StatusCodeError: 400 - {"error":{"message":"(#100) No permission to publish the video","type":"OAuthException","code":100,"fbtrace_id":"ABYgvmb1bYPROFeYGFlj72P"}}

**Comment by jonnymangothinkjam** [comment]
these were the graph API permissions set if it helps too

![image](https://user-images.githubusercontent.com/25662069/82345468-32fd9580-99ed-11ea-8d67-c1383f000aa2.png)

**Comment by jonnymangothinkjam** [comment]
sorry, closing as i figured out the issue - the permissions linking wasnt set up right, my bad

**Comment by mrdotb** [comment]
I made it work again after a bit of struggle with the multiples kind of tokens.
I needed to extend my token to a long lived token in order upload video.
In my case I upload on a facebook page.

![Selection_002](https://user-images.githubusercontent.com/7212914/82347696-26dffb00-9a22-11ea-915b-d131cbcca745.png)
![Selection_001](https://user-images.githubusercontent.com/7212914/82347705-29425500-9a22-11ea-8f4e-9463b9d49596.png)
